client.go

```diff
@@ -31,8 +31,8 @@ const (
 	defaultRepositoryConfigPath = "/tmp/.helmrepo"
 )
 
-// New returns a new Helm client with the provided options
-func New(options *Options) (*Client, error) {
+// NewClient returns a new Helm client with the provided options
+func NewClient(options *Options) (*HelmClient, error) {
```

Comment:
Why the rename from `New` to `NewClient`? Now, this would be called with `helmclient.NewClient(...)`, which stutters a bit more than the previous `helmclient.New(...)`.

Reply:
My thought here was to have a more meaningful name for the function, if (eventually) a fake client is introduced to the package. 
So, when the fake client is introduced, it's constructor method would be called `NewFakeClient` instead of `NewFake`. Yet i have to agree, putting this into perspective with the package name, `helmclient.New` / `helmclient.NewFake` should be talkative enough :+1: